jaxws-ri/runtime/rt/src/main/java/com/sun/xml/ws/fault/SOAPFaultBuilder.java

```diff
@@ -579,4 +587,9 @@ private static JAXBContext createJAXBContext() {
    public static boolean isCaptureExceptionMessage() {
         return captureExceptionMessage;
    }
+
+
+    private static String setFaultStringToServerError(String faultString) {
```

Comment:
it would be awesome to follow existing patterns and general java conventions in the class - this method does not set any variable, so its name should not start with _set_.

Reply:
done